docs/reference/release-notes/highlights-7.0.0.asciidoc

```diff
@@ -298,6 +298,25 @@ https://www.elastic.co/blog/faster-retrieval-of-top-hits-in-elasticsearch-with-b
 WAND: Faster Retrieval of Top Hits in Elasticsearch].
 //end::notable-highlights[]
 
+//tag::notable-highlights[]
+[float]
+=== Faster Boosting with Feature Fields
```

Comment:
The other sections seem to use sentence style capitalization, so this should match:
```suggestion
=== Faster boosting with feature fields
```